src/transformState.js

```diff
@@ -5,7 +5,45 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  const ADD_PROPERTIES = 'addProperties';
+  const REMOVE_PROPERTIES = 'removeProperties';
+  const CLEAR = 'clear';
```

Comment:
I'm not sure about this is helpful because you used this variables only 1 time each)